core/src/main/java/me/pikamug/quests/dependencies/BukkitDependencies.java

```diff
@@ -217,8 +218,27 @@ public void linkZnpcsPlus() {
                     plugin.getLogger().info("Successfully linked Quests with ZNPCsPlus "
                             + znpcsPlus.getDescription().getVersion());
                 }
-            } catch (final Exception e) {
-                plugin.getLogger().warning("Incompatible version of ZNPCsPlus found. ZNPCsPlus in Quests not enabled.");
+            } catch (Exception ignored) {
```

Comment:
Why ignore this exception?

Reply:
In that case, my last commit 2e6ab6c seems obsolete, as `linkZnpcsPlus` mostly just ensures that a compatible version is present. I'll take care of it.

Does ZNPCsPlus 2.x guarantee that `NpcApiProvider.get()` will always return the NpcApi, or can it return empty/null?

Reply:
> Does ZNPCsPlus 2.x guarantee that `NpcApiProvider.get()` will always return the NpcApi, or can it return empty/null?

It can return null if ZNPCsPlus was not enabled due to some reason or loaded after the Quests plugin.
as Quests checks if the plugin is enabled or not, this should not be a problem. And ZNPCsPlus is always loaded before Quests cause......
![image](https://github.com/PikaMug/Quests/assets/49519439/4dbfa88d-9d3e-4f86-adf2-04defbc02289)

Reply:
and yeah @PikaMug , the last commit you did was still causing `NoClassDefFound` because you were trying to cast `lol.pyr.znpcsplus.ZNpcsPlus` to `lol.pyr.znpcsplus.ZNPCsPlus`. I think you forgot that `ZNPCsPlus` was changed to `ZNpcsPlus` in https://github.com/Pyrbu/ZNPCsPlus/pull/39